plugins/ecs-bridge/main.go

```diff
@@ -47,8 +47,7 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```

Comment:
You can definitely retrieve `stderr` from processes launched by Go.  I'd prefer to have errors on `stderr`.